requirements/dev.txt

```diff
@@ -154,7 +154,7 @@ pep517==0.13.0
     # via
     #   -r requirements/pip-tools.txt
     #   build
-pip-tools==6.8.0
+pip-tools==6.9.0
```

Comment:
No breaking changes in the new version: https://github.com/jazzband/pip-tools/blob/master/CHANGELOG.md#690-2022-10-05